### DISCUSSION TITLE
Hub Needed?

### DISCUSSION THREAD

**Discussion by yshahin** [RobertD502/home-assistant-flair#79] [Q&A] [answered]
Is the hub needed to use Flair?
Can we connect directly to the devices?
What protocol are they using?

**Comment by RobertD502** [comment, marked as answer]
1) Need at least one Flair Bridge or Flair Puck (a Flair Bridge has greater coverage than a single Flair Puck so it makes sense to go with the Flair Bridge if you don't need the features of the Puck) to act as a gateway between Flair vents and Flair servers.

2) No (not with this integration at least). To do that you'd have to get or build a DASH7 gateway and see if you can get the vents to connect to this gateway (this is a simplified explanation).

3) 915 MHz - likely DASH7 based on my findings a while back.

For other questions, I'd recommend contacting their sales support directly as I'm not a Flair employee and they'd be able to help you out more.

**Reply by holocronology** [reply]
>Need at least one Flair Bridge or Flair Puck (a Flair Bridge has greater coverage than a single Flair Puck so it makes sense to go with the Flair Bridge if you don't need the features of the Puck) to act as a gateway between Flair vents and Flair servers.

I'm newly coming to this--both flair devices and home assistant. I can use this to connect to the Flair Pucks without a Flair Bridge, if I'm understanding this statement correctly, yes?

I can connect to multiple Pucks with this and without a bridge?

The connecting HA via a bridge offers reduced HA functionality?

**Reply by RobertD502** [reply]
> I can use this to connect to the Flair Pucks without a Flair Bridge, if I'm understanding this statement correctly, yes?
> 
> I can connect to multiple Pucks with this and without a bridge?

The integration doesn't connect directly to any of the Flair devices since there is no local API. The integration uses their cloud-based API to speak with Flair's servers.

> The connecting HA via a bridge offers reduced HA functionality?

It isn't so much about Home Assistant functionality. It comes down to what it is you're trying to accomplish - are you just trying to control vents, do you want Flair to handle opening / closing vents or do you want that to be something you automate yourself from within Home Assistant, etc.

**Reply by holocronology** [reply]
> It isn't so much about Home Assistant functionality. It comes down to what it is you're trying to accomplish - are you just trying to control vents, do you want Flair to handle opening / closing vents or do you want that to be something you automate yourself from within Home Assistant, etc.

Thanks for the reply. I do actually have HA running, but I haven't gotten into it deeply. I keep most of my home automation inside Hubitat. That said, here is my potential use case:

I am converting our house to mini-split heat pumps. I am considering Flair Pucks for more user friendly controls. This would also allow me app access to all devices.

However, I would prefer to have a central location where I can see everything and, more importantly, the rest of the household can go an have options such as, Set house to X, set upstairs to X, set gues room to X, etc...unfortunately, there isn't any integration that's been developed currently available on Hubitat. So this leads me to HA and here. 

I will primarily be using Pucks, but may decide to add some vents as "transfer vents" between certain rooms.

**Reply by RobertD502** [reply]
In that case, Pucks is the way to go since the bridge lacks the IR transmitter needed to control mini splits.

**Reply by holocronology** [reply]
> In that case, Pucks is the way to go since the bridge lacks the IR transmitter needed to control mini splits.

Indeed, however the bridge does allow for "battery" based powering of the Puck. Does having the bridge impact functionality in HA integration?

**Reply by RobertD502** [reply]
> > In that case, Pucks is the way to go since the bridge lacks the IR transmitter needed to control mini splits.
> 
> 
> 
> Indeed, however the bridge does allow for "battery" based powering of the Puck. Does having the bridge impact functionality in HA integration?

You can have one puck that is hardwired acting as the gateway and the other pucks being powered by battery or you can use a bridge and have all pucks on batteries.

Functionality for a puck: Can act as a gateway to Flair servers, IR for mini split control, has temp and light sensor.

Functionality of bridge: Acts as a gateway to Flair servers.

**Comment by dewbot6** [comment]
Hi All, been working on decoding the RF protocol if anyone is interested:
https://github.com/dewbot6/flair-vent-rf
